### ISSUE TITLE
examples/node_pool statically defined the pod_range name for pool-03

### ISSUE THREAD

**Issue by apeabody** [terraform-google-modules/terraform-google-kubernetes-engine#1222]
### TL;DR

https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/77d08e02d738c38a5c68987572ef506a987dd41e/examples/node_pool/main.tf#L76

### Terraform Configuration

```hcl
N/A
```

### Terraform Version

```sh
N/A
```

**Comment by morgante** [comment]
@apeabody We really want to *reduce* the use of variables over-time, actually. Ideally most examples should be self-contained and liberally include static values instead of variables.

**Comment by apeabody** [comment]
@morgante Agreed, this issue is the example isn't _totally_ self-contained as that network range is defined in the test fixture.  Thought?

**Comment by morgante** [comment]
The ideal fix would be moving the network creation from the fixture to the example. Eventually we want to get rid of fixtures entirely.

**Comment by apeabody** [comment]
SG - I updated the issue to align.
